.github/workflows/script/sw_build_and_push_gem.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+GEM_NAME="$1"
+
+cd "instrumentation/$GEM_NAME"
```

Comment:
I recommend running this script through `shellcheck` to lint the bash.

Reply:
updated with lint for shell

Reply:
Thanks!